src/com/sfuentes/OnlyPositive.java

```diff
@@ -0,0 +1,18 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class OnlyPositive {
+  public ArrayList<Double> showOnlyPositiveNumbers(double[] numbers) {
```

Comment:
Here you can rename this method to filterPositiveNumbers. This because you're not showing anything in this method.

Reply:
Fixed it